src/Executor/Executor.php

```diff
@@ -65,6 +65,8 @@ public function createRuntime(
         string $source,
         string $image,
         string $version,
+        int $cpus = 1,
+        int $memory = 512,
```

Comment:
I dont see those two being used.

Reply:
Yeah since I accidentally deleted the part that used it, adding it back now